PythonAPI/docs/world.yml

```diff
@@ -0,0 +1,414 @@
+    # - DESCRIPTION ------------------------
+    doc: >
+      Class that defines the attachment options.
+    # - PROPERTIES -------------------------
+    instance_variables:
+    - var_name: Rigid
+      doc: >
+    - var_name: SpringArm
+      doc: >
+    # --------------------------------------
+
+  - class_name: World
+    # - DESCRIPTION ------------------------
+    doc: >
+    # - PROPERTIES -------------------------
+    instance_variables:
+    - var_name: id
+      type: int
+      doc: >
+        The id of the episode associated with this world.
+    - var_name: debug
+      type: carla.DebugHelper
+      doc: >
+    # - METHODS ----------------------------
+    methods:
+    - def_name: get_blueprint_library
+      return: carla.BlueprintLibrary
+      doc: >
+        Return the list of blueprints available in this world. These blueprints
+        can be used to spawn actors into the world.
+    # --------------------------------------
+    - def_name: get_map
+      return: carla.Map
+      doc: >
+        Return the map that describes this world.
+    # --------------------------------------
+    - def_name: get_spectator
+      return: carla.Actor
+      doc: >
+        Return the spectator actor. The spectator controls the view in the
+        simulator window.
+    # --------------------------------------
+    - def_name: get_settings
+      return: carla.WorldSettings
+      doc: >
+    # --------------------------------------
+    - def_name: apply_settings
+      return: int
+      params:
+      - param_name: world_settings
+        type: carla.WorldSettings
+      doc: >
+        Returns the id of the frame when the settings took effect.
+    # --------------------------------------
+    - def_name: get_weather
+      return: carla.WeatherParameters
+      doc: >
+        Retrieve the weather parameters currently active in the world.
+    # --------------------------------------
+    - def_name: set_weather
+      params: 
+      - param_name: weather
+        type: carla.WeatherParameters
+      doc: >
+        Change the weather in the simulation.
+    # --------------------------------------
+    - def_name: get_snapshot
+      return: carla.WorldSnapshot
+      doc: >
+        Return a snapshot of the world at this moment.
+    # --------------------------------------
+    - def_name: get_actor
+      return: carla.Actor
+      params:
+      - param_name: actor_id
+        type: int
+      doc: >
+        Find actor by id, return nullptr if not found.
+    # --------------------------------------
+    - def_name: get_actors
+      return: carla.ActorList
+      doc: >
+        Return a list with all the actors currently present in the world.
+    # --------------------------------------
+    - def_name: get_actors
+      return: carla.ActorList
+      params: 
+      - param_name: actor_ids
+        type: list(int)
+      doc: >
+        Return a list with the actors requested by ActorId.
```

Comment:
I would merge both versions of `get_actors()`, saying that by default it retrieves every actor.